<launch>
    <arg name="sim_wind_x" default="-2" />
    <arg name="sim_wind_y" default="0" />

    <param name="robot_description" command="cat $(find sailbot_sim)/urdf/boat.urdf" />
    <node name="sim_odom" pkg="sailbot_sim" type="sim_odom.py" />
    <node name="wind_markers" pkg="sailbot_sim" type="wind_markers.py" />

    <node name="true_wind" pkg="sensors" type="truewind_simple.py" />
    <node pkg="tf" type="static_transform_publisher" name="sail_tf_broadcaster" args="0 0 0 0.7895 0 0 boat sail 100" />

    <param name="/sim_wind_vector/x" type="double" value="$(arg sim_wind_x)" />
    <param name="/sim_wind_vector/y" type="double" value="$(arg sim_wind_y)" />
</launch>
